apps/fz_common/lib/fz_net.ex

```diff
@@ -65,4 +65,12 @@ defmodule FzCommon.FzNet do
   def valid_hostname?(hostname) when is_binary(hostname) do
     String.match?(hostname, @host_regex)
   end
+
+  def to_complete_url(str) when is_binary(str) do
+    case URI.parse(str) do
+      %{host: nil, scheme: nil} -> "https://" <> str
+      %{scheme: nil} -> "https" <> str
```

Comment:
Curious when this happens?

Reply:
In an edge case a user types `//hostname`

Reply:
Oh just did some testing, caddy doesn't support that so will remove it